virt/virt-4-10-release-notes.adoc

```diff
@@ -0,0 +1,228 @@
+
+* You can now xref:../virt/virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc#virt-hot-plugging-virtual-disks[hot-plug and hot-unplug virtual disks] when you want to add or remove them from your virtual machine without stopping the virtual machine instance.
+
+* You can now use the Red Hat Enterprise Linux 9 Alpha template to create virtual machines.
+
+* You can now link:https://access.redhat.com/articles/6409731[deploy {VirtProductName} on AWS bare metal nodes].
+
+[id="virt-4-10-bug-fixes"]
+== Bug fixes
+
+* The *Template provider* menu in the web console no longer offers "Red Hat Supported" as a template search filter, to avoid confusion with the "Red Hat Provided" filter.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1952737[*BZ#1952737*])
+
+* The KubeVirt plugin now checks the API version available and uses the correct version, rather than defaulting to the v1 API version, which resulted in an API mismatch and prevented virtual machine creation.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*]), (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979114[*BZ#1979114*])
+
+* The {op-system-base-full} 6 template is no longer prioritized in the web console.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1978200[*BZ#1978200*])
+
+* The {op-system-base-full} 6 template is no longer labeled as a community-provided template in the web console.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1978202[*BZ#1978202*])
+
+* The web console can now retrieve more information from virtual machines, including time zone and number of active users.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1979190[*BZ#1979190*])
+
+* Live migration between nodes with incompatible CPUs is now prevented on clusters containing nodes that are not configured identically. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760028[*BZ#1760028*])
+
+* If you initially deployed {VirtProductName} version 2.4.z or earlier, you can now upgrade to the latest version without using a workaround. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986989[*BZ#1986989*])
+
+* If you run {VirtProductName} 2.6.5 with  {product-title} 4.8 or later, you can now create a virtual machine from the *Customize* wizard. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
+
+* RHV VM import no longer fails if the VM affinity policy is set to `Migratable` rather than `Pinned`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977277[*BZ#1977277*])
+
+* Selecting *Create* -> *With Import wizard* on the *Virtualization* page of the {VirtProductName} web console no longer results in an erroneous error message. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974812[*BZ#1974812*])
+
+[id="virt-4-10-known-issues"]
+== Known issues
+
```

Comment:
It's a good idea to add a commented-out note dividing the new and old Known Issues so that they don't get mixed up while you're figuring out if the old ones still apply to the new release.